src/js/pages/services/DeploymentsTab.js

```diff
@@ -1,11 +1,73 @@
+import moment from 'moment';
 import React from 'react';
+import {Table} from 'reactjs-components';
 
 import AlertPanel from '../../components/AlertPanel';
+import EventTypes from '../../constants/EventTypes';
+import MarathonStore from '../../stores/MarathonStore';
+import ResourceTableUtil from '../../utils/ResourceTableUtil';
+import StringUtil from '../../utils/StringUtil'
+
+const columnClassNameGetter = ResourceTableUtil.getClassName;
+const columnHeading = ResourceTableUtil.renderHeading({
+  id: 'DEPLOYMENT',
+  startTime: 'STARTED',
+  status: 'STATUS'
+});
+const columns = [
+  {
+    className: columnClassNameGetter,
+    heading: columnHeading,
+    prop: 'id',
+    render: function (prop, deployment) {
+      return deployment.getId();
+    }
+  },
+  {
+    className: columnClassNameGetter,
+    heading: columnHeading,
+    prop: 'startTime',
+    render: function (prop, deployment) {
+      return moment(deployment.getStartTime()).fromNow();
+    }
+  },
+  {
+    className: columnClassNameGetter,
+    heading: columnHeading,
+    prop: 'status',
+    render: function (prop, deployment) {
+      return `Step ${deployment.getCurrentStep()} of ${deployment.getTotalSteps()}`;
+    }
+  }
+];
 
 class DeploymentsTab extends React.Component {
 
   constructor() {
     super(...arguments);
+    this.onDeploymentsChange = this.onDeploymentsChange.bind(this);
+
+    this.state = {
+      deployments: MarathonStore.get('deployments')
```

Comment:
I find it quite unsettling that we're assigning the Deployments List by reference to our state – we have no protection from external mutations. That's a generic issue with our Structs approach, not a problem in this PR per se.